src/query/storages/fuse/fuse/src/fuse_table.rs

```diff
@@ -149,6 +150,29 @@ impl FuseTable {
         Ok(table_storage_prefix(db_id, table_id))
     }
 
+    pub fn table_snapshot_statistics_format_version(&self, location: &String) -> Result<u64> {
+        Ok(TableMetaLocationGenerator::snapshot_version(location))
```

Comment:
is `snapshot_statistics_version` supposed to be used here?

Reply:
yes, in `read_table_snapshot_statistics`